meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -312,6 +312,7 @@ MACHINE_FIRMWARE_mx8mm = "linux-firmware-imx-sdma-imx7d"
 WKS_FILE_mx8mm_sota = "sdimage-imx8-sota.wks.in"
 ## iMX8MM EVK
 MACHINE_FEATURES_remove_imx8mmevk = "qca9377 qca6174"
+IMXBOOT_TARGETS_mx8mm = "flash_spl_signed"
```

Comment:
Can we move this setting up under the `*_mx8mm` settings?

Reply:
yes